[2022-07-01 15:22:07,798] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:12:00+00:00 [queued]>
[2022-07-01 15:22:07,801] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:12:00+00:00 [queued]>
[2022-07-01 15:22:07,801] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-01 15:22:07,801] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-01 15:22:07,802] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-01 15:22:07,819] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-30 21:12:00+00:00
[2022-07-01 15:22:07,823] {standard_task_runner.py:52} INFO - Started process 18523 to run task
[2022-07-01 15:22:07,831] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-30T21:12:00+00:00', '--job-id', '1155', '--raw', '--subdir', 'DAGS_FOLDER/CarPriceDag.py', '--cfg-path', '/tmp/tmp8lrdekd8', '--error-file', '/tmp/tmpdmux5r5r']
[2022-07-01 15:22:07,831] {standard_task_runner.py:80} INFO - Job 1155: Subtask save_data_task
[2022-07-01 15:22:07,868] {task_command.py:369} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:12:00+00:00 [running]> on host DESKTOP-9B44JSS.localdomain
[2022-07-01 15:22:08,005] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-30T21:12:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-30T21:12:00+00:00
[2022-07-01 15:22:08,222] {python.py:173} INFO - Done. Returned value was: None
[2022-07-01 15:22:08,230] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220630T211200, start_date=20220701T132207, end_date=20220701T132208
[2022-07-01 15:22:08,283] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-01 15:22:08,304] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
